Tests/GRPCHTTP2CoreTests/GRPCMessageDeframerTests.swift

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import NIOCore
+import XCTest
+
+@testable import GRPCHTTP2Core
+
+final class GRPCMessageDeframerTests: XCTestCase {
```

Comment:
FYI NIOTestUtils has a helper we should use: https://github.com/apple/swift-nio/blob/635b2589494c97e48c62514bc8b37ced762e0a62/Sources/NIOTestUtils/ByteToMessageDecoderVerifier.swift#L54